pkg/controller/controller.go

```diff
@@ -360,5 +360,17 @@ func GetK8sEvents(configFile string, event string, namespace string, name string
 	message = NamespaceGet(api, "", name)
 	fmt.Printf("NAMESPACE API: List of all namespace retrieved %s", message)
      }
+     if (strings.ToLower(event) == "service"){
+		message = ServiceGet(api, namespace, name)
```

Comment:
@SaravanakumarR2018 , Make  this function aligned with tab. Correct the above if condition with tabs